### DISCUSSION TITLE
Is HTML-formatting possible in a custom view?

### DISCUSSION THREAD

**Discussion by iliakan** [mitmproxy/mitmproxy#6691] [General]
Hello,

Is there a way to apply any HTML-formatting to a custom view of a request/response body?
Such as using HTML-tags `<b>`, etc.

Or is there any other formatting option?

**Comment by mhils** [comment]
The best we have right now is colorization, the [JSON contentview](https://github.com/mitmproxy/mitmproxy/blob/main/mitmproxy/contentviews/json.py) is an example of that. HTML formatting is unlikely to arrive because we also want to support the console TUI. But the contentview API needs an overhaul for sure. :)
